extensions/roc-plugin-test-jest/src/config/roc.config.js

```diff
@@ -0,0 +1,14 @@
+export default {
+    settings: {
+        test: {
+            jest: {
+                config: {
+                },
+                reporter: {
```

Comment:
I'm wondering if this maybe should not be coupled to Jest in this way but rather more generic.